app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,34 @@
+class ManageIQ::Providers::IbmCloud::VPC::CloudManager::EventCatcher::Stream
+  attr_reader :ems, :stop_polling, :poll_sleep
+
+  def initialize(ems, options = {})
+    @ems = ems
+    @stop_polling = false
+    @from = DateTime.current.to_time.to_i.to_s
```

Comment:
Maybe not for this PR, if this was copied from others, but why are we using `DateTime.current.to_time` (which comes from ActiveSupport), as opposed to `Time.now` (which is in Ruby core)?  The latter is simpler and gives the same number.

cc @agrare

Reply:
```ruby
[6] pry(main)> [DateTime.current.to_time.to_i, Time.now.to_i]
=> [1627569302, 1627569302]
```

Reply:
Separate question, more for @agrare, but do we have a way to record the last "key" that we ran against?  The reason I'm asking is that if the event catcher goes down, we will lose everything during the down time.  I thought we had a way to record the last "key" that we saw, so we can pick up where we left off.

If not, maybe we should take the last event that was recorded in the table and use that timestamp?

Reply:
Ah didn't know there was a simpler way to do it, thank you

Reply:
@Fryguy if the API is expecting the start point to be timestamp based then we can do `ems.ems_events.order(:timestamp).last`